src/components/WorkQueueStatusIcon.vue

```diff
@@ -1,31 +1,33 @@
 <template>
```

Comment:
Before we merge this I'd double check that this component isn't used elsewhere - as written this will no longer work for deprecated work queues (not sure that matters but worth a callout)

Reply:
If this is being used elsewhere I'd revert these changes and create a new component specifically for work pool queues - if not, please rename this component to avoid confusion

Reply:
Ahhh... Good call out thank you.  Let me check!

Reply:
I'm not sure those components are actively being used but I've copied the updates to a new WorkPoolQueueStatusBadge and WorkPoolQueueStatusIcon components and reverted the WorkQueueStatusBadge, WorkQueueStatusIcon, WorkQueueDetails and WorkQueuesTable components.  This should give us cover if there are any old work queues out there and make it easier to remove those deprecated components when we think we can.

Reply:
Thank you @zhen0!!